aws/resource_aws_emr_cluster.go

```diff
@@ -25,6 +25,10 @@ func resourceAwsEMRCluster() *schema.Resource {
 		Read:   resourceAwsEMRClusterRead,
 		Update: resourceAwsEMRClusterUpdate,
 		Delete: resourceAwsEMRClusterDelete,
+		Importer: &schema.ResourceImporter{
```

Comment:
This can be acceptance tested by adding the following as a `TestStep` e.g. after the existing step of `TestAccAWSEMRCluster_basic`:

```go
{
	ResourceName:      "aws_emr_cluster.tf-test-cluster",
	ImportState:       true,
	ImportStateVerify: true,
},
```

Reply:
Done. Thanks for the tip!